### ISSUE TITLE
Problems installing ASReview -

### ISSUE THREAD

**Issue by YNoteboom** [asreview/asreview#1410]
**Describe the bug**
After installing python and ASReview I get the following error during installing ASreview:

WARNING: The script asreview.exe is installed in 'C:\Users\VNOB-0716\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.

**To Reproduce**
Unistalled ASreview and installed it again but same error remains. Try to open ASreviw via the folder of python but that didnt work either. 

**Screenshots**

![image](https://user-images.githubusercontent.com/130556092/233323606-a998ec0b-515a-4dca-9fab-b9d727166899.png)

**Version information**
Windows
- python 3.11.3
 - ASReview 1.0

**Additional context**
Had some problem with pip before but did manage to fix that.

**Comment by rohitgarud** [comment]
Hi @YNoteboom. Try adding `C:\Users\VNOB-0716\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310` to your 'PATH' in environment variables

**Comment by Rensvandeschoot** [comment]
Have you tried the solution offered here: https://asreview.readthedocs.io/en/latest/troubleshooting.html#unknown-command-pip?

**Comment by YNoteboom** [comment]
Yes, thank you so much. Did not found the 'read the docs' but will use in future, very helpfull! 
This was the solution:

python -m asreview lab
